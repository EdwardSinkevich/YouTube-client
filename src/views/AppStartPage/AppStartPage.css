@font-face {
  font-family: "FontAwesome";
  src: url("https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/fonts/fontawesome-webfont.woff2?v=4.3.0")
      format("woff2"),
    url("https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/fonts/fontawesome-webfont.woff?v=4.3.0")
      format("woff"),
    url("https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/fonts/fontawesome-webfont.ttf?v=4.3.0")
      format("truetype"),
    url("https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/fonts/fontawesome-webfont.svg?v=4.3.0#fontawesomeregular")
      format("svg");
}

* {
  margin: 0;
  padding: 0;
  border: 0;
  outline: 0;
  box-sizing: border-box;
}

body {
  min-width: 320px;
  font-family: "Arial", sans-serif;
  background: #343d46;
}

.container {
  overflow-x: hidden;
}

.search-wrapper {
  display: flex;
  justify-content: center;
}

.search-box {
  width: 60%;
  height: 50px;
  margin-top: 10px;
  position: relative;
}

.button {
  width: 40px;
  height: 100%;
  background: none;
  position: absolute;
  cursor: pointer;
}

.button::before {
  font-family: "FontAwesome";
  content: "\f002";
  font-size: 30px;
  padding: 0 10px;
}

.search-input {
  width: 100%;
  height: 100%;
  padding-left: 50px;
  border-radius: 6px;
  font-size: 1.4rem;
}
