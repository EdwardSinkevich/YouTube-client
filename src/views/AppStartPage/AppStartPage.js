import './AppStartPage.css';

export default class AppStartPage {
  static render() {
    document.body.innerHTML = `<div class='container'><div class='search-wrapper'>
    <div class='search-box'><button class='button'></button>
    <input class='search-input'></div></div><ul class='clips'></ul>
    <div class='controls'><button class='control-button first-page'><span class="tooltip"></span></button>
    <button class='control-button prev-page'><span class="tooltip"></span></button>
    <button class='control-button current-page'></button>
    <button class='control-button next-page'><span class="tooltip"></span></button></div></div>`;
  }
}
