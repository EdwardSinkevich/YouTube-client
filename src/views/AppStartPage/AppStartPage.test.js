import AppStartPage from './AppStartPage';

describe('AppStartPage.render', () => {
  it('Should be render correctly', () => {
    AppStartPage.render();
    expect(document.querySelector('.clips').innerHTML).toMatchSnapshot();
  });
});
