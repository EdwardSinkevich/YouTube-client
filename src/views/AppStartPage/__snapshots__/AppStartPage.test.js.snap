// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppStartPage.render Should be render correctly 1`] = `""`;
